### ISSUE TITLE
dragonflybsd5 no sudoer

### ISSUE THREAD

**Issue by zhanhb** [lavabit/robox#227]
```
We trust you have received the usual lecture from the local System
Administrator. It usually boils down to these three things:

    #1) Respect the privacy of others.
    #2) Think before you type.
    #3) With great power comes great responsibility.

vagrant is not in the sudoers file.  This incident will be reported.
```
version: 3.6.4
provider: virtualbox

**Comment by ladar** [comment]
Fixed.

**Comment by ladar** [comment]
I should note that the fix probably won't be in 3.6.6 since those boxes have already been built (they're just waiting for me to check and upload em). So this will be in the next, 3.6.8 upload.
